docker/package/tezos_baking_wizard.py

```diff
@@ -497,6 +497,15 @@ def get_tezos_client_options(self):
             + self.config["node_rpc_addr"]
         )
 
+    def get_current_head_level(self):
+        output = get_proc_output(
+            "sudo -u tezos tezos-client "
+            + self.get_tezos_client_options()
+            + " rpc get /chains/main/blocks/head/header"
+        ).stdout
+
+        return re.search(b'"level": ([0-9]+),', output).group(1).decode()
+
```

Comment:
why not Python to invoke rpc directly (urllib.request is already in use), or at least parse json response properly (not with regex)?

Reply:
Indeed, nice suggestion